.travis.yml

```diff
@@ -16,3 +16,5 @@ git:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("StatsBase"); Pkg.test("StatsBase"; coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("StatsBase")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'cd(Pkg.dir("StatsBase")); include(joinpath("docs", "make.jl"))'
```

Comment:
Yes, I prefer that form, but for some reason the Documenter manual uses two separate lines.